src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -376,7 +379,10 @@ void handleExplosion(Location location, Entity entity, List<Block> blocks)
             //if no, then also consider surface rules
             if (claim == null)
             {
-                if (!applySurfaceRules || block.getLocation().getBlockY() < GriefPrevention.instance.getSeaLevel(world) - 7)
+                //if the explosion happens above sea level, don't explode the block.
+                if (location.getBlockY() >= seaLevel) continue;
```

Comment:
Um, what about those who disable surface rules? Or am I missing something here?

Reply:
haha fair, not sure why that got removed ^^